pipeline/trusted-resources/pkg/trustedtask/trustedtaskrun_test.go

```diff
@@ -394,9 +386,27 @@ func getSignedTask(unsigned *v1beta1.Task, signer signature.Signer) (*v1beta1.Ta
 	return signedTask, nil
 }
 
-// Generate key files to tmpdir, set configMap and return signer
-func getSignerFromFile(t *testing.T, ctx context.Context, k8sclient kubernetes.Interface) (signature.Signer, error) {
+func setupContext(ctx context.Context, k8sclient kubernetes.Interface, secretpath string) context.Context {
+	cmw := informer.NewInformedWatcher(k8sclient, system.Namespace())
+	store := config.NewConfigStore(logging.FromContext(ctx).Named("config-store"))
+	store.WatchConfigs(cmw)
+	cm := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: nameSpace,
+			Name:      signingConfigMap,
+		},
+		Data: map[string]string{"signing-secret-path": secretpath},
```

Comment:
we've already defined it as a const variable `cosignKey`?

Reply:
yes, thanks!